src/lib/serviceRoutinesV2/logLevelTreat.cpp

```diff
@@ -55,11 +55,44 @@ std::string changeLogLevel
   std::string  level   = ciP->uriParam[URI_PARAM_LEVEL];
   const char*  levelP  = level.c_str();
 
-  if (level.empty())
+  std::string logSize     = ciP->uriParam[URI_PARAM_LOG_SIZE];
+  int logSizeValue        = atoi(logSize.c_str());
+  std::string payloadSize = ciP->uriParam[URI_PARAM_PAYLOAD_SIZE];
+  int payloadSizeValue    = atoi(payloadSize.c_str());
+
+  if (level.empty() && logSize.empty() && payloadSize.empty())
   {
     ciP->httpStatusCode = SccBadRequest;
-    alarmMgr.badInput(clientIp, "no log level in URI param");
-    return "{\"error\":\"log level missing\"}";
+    alarmMgr.badInput(clientIp, "no log configs in URI param");
+    return "{\"error\":\"log config missing\"}";
+  }
+
+  if (!logSize.empty())
+  {
+    if (logSizeValue > 0)
+    {
+      logLineMaxSize = logSizeValue;
+    }
+    else
+    {
+      ciP->httpStatusCode = SccBadRequest;
+      alarmMgr.badInput(clientIp, "invalid logLineMaxSize in URI param", logSize);
+      return "{\"error\":\"invalid logLineMaxSize, logLine size should be > 0\"}";
```

Comment:
```suggestion
      return "{\"error\":\"invalid lineMaxSize, lineMaxSieze should be > 0\"}";
```

To align with the name of the field that provides this information in the PUT request.

Reply:
Fixed in [24de8d9](https://github.com/telefonicaid/fiware-orion/pull/4289/commits/24de8d9e544ab1ddba1af46cdb70ded49bb85e2e)